commonutils/publish.gradle

```diff
@@ -0,0 +1,81 @@
+apply plugin: 'maven-publish'
+
+ext.localProperties = new Properties()
+ext.localProperties.load(project.rootProject.file('local.properties').newDataInputStream())
+
+ext.getLocalProperty = { propertyName ->
+    def property = localProperties.getProperty(propertyName)
+    return property == null ? "" : property
+}
+
+ext.artifactPath = "build/outputs/aar/commonutils-release.aar"
```

Comment:
Is this 'hard-coded' path robust enough? Could there be an already existing variable/field holding (part of) this path.